HTML/mockup-minimo/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inconsolata|Playfair+Display|Ubuntu:300,400&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Mockup 3</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
```

Comment:
only menu should be in `nav` tag as logo generally is not a navigation, you can use for header class header, after have elements like header__logo/header__menu-icon, your menu would be part of nav